xarray/coding/cftimeindex.py

```diff
@@ -59,10 +60,11 @@
 REPR_ELLIPSIS_SHOW_ITEMS_FRONT_END = 10
 
 
-if LooseVersion(pd.__version__) > LooseVersion("1.2.3"):
-    OUT_OF_BOUNDS_TIMEDELTA_ERROR = pd.errors.OutOfBoundsTimedelta
+OUT_OF_BOUNDS_TIMEDELTA_ERRORS: Tuple[Type[Exception], ...]
+if LooseVersion(pd.__version__) > LooseVersion("1.1.0"):
+    OUT_OF_BOUNDS_TIMEDELTA_ERRORS = (pd.errors.OutOfBoundsTimedelta, OverflowError)
 else:
-    OUT_OF_BOUNDS_TIMEDELTA_ERROR = OverflowError
+    OUT_OF_BOUNDS_TIMEDELTA_ERRORS = (OverflowError,)
```

Comment:
not sure if this really is better, but I would have used:
```suggestion
try:
    OUT_OF_BOUNDS_TIMEDELTA_ERRORS = (pd.errors.OutOfBoundsTimedelta, OverflowError)
except AttributeError:
    OUT_OF_BOUNDS_TIMEDELTA_ERRORS = (OverflowError,)
```

Reply:
Thanks!  I do like your approach better.  I updated things accordingly.